storybookjs/storybook
# Switch to monorepo

Issue: #742 

## What I did
Moved all the source for this package into a /packages/react-storybook
Merged in the monorepo master
Resolved mergeconflicts


## How to test
install dependencies - this should bootstrap lerna.
run `npm run test` in the root
